test/asynctest.c

```diff
@@ -413,6 +415,51 @@ static int test_ASYNC_start_job_ex(void)
     return ret;
 }
 
+static void *test_alloc_stack(size_t *num)
+{
+    custom_alloc_used = 1;
+    return malloc(*num);
```

Comment:
Shouldn't this just be `OPENSSL_malloc` and `OPENSSL_free`?

Reply:
Sure, it was more a demonstration of using non-OpenSSL malloc/free functions like a real world application would do, but it doesn't really matter... Switched to `OPENSSL_malloc` and `OPENSSL_free`.